src/extension/ncwms/pom.xml

```diff
@@ -64,6 +64,17 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
```

Comment:
Geoserver uses

        <dependency> 
            <groupId>net.sf.json-lib</groupId> 
            <artifactId>json-lib</artifactId> 
            <classifier>jdk15</classifier>
        </dependency>

Looks compatible with the way you are doing it now except parsing is more succinct e.g.

    JSONObject.fromObject(readUrl(metadataUrl))